checks/http/google.rb

```diff
@@ -125,6 +125,19 @@ def generate_checks(url)
           :paths => ["#{url}"],
           :inference => false
         },
+        {
+          :type => "fingerprint",
+          :category => "application",
+          :tags => ["Web Server", "Hosting"],
+          :vendor => "Google",
+          :product =>"Google Web Server",
+          :match_details =>"Server header",
+          :version => nil,
+          :match_type => :content_headers,
+          :match_content =>  /server: gws/i,
```

Comment:
can tighten this regex up bit. - add ^ and $ to ensure that this only matches something with exactly this string